packages/collector/test/tracing/protocols/http/native_fetch/test.js

```diff
@@ -689,7 +724,8 @@ function verifyHttpExit({
   withClientError,
   withServerError,
   withTimeout,
-  serverControls
+  serverControls,
+  params = null
```

Comment:
🤔

Reply:
I added another optional check.
If the test passes params, we test params.